examples/ct/ct_handlers_test.go

```diff
@@ -185,14 +218,14 @@ func TestPostHandlersAcceptNonEmptyCertChain(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		// For now they return not implemented as the handler is a stub
-		assert.Equal(t, http.StatusNotImplemented, resp.StatusCode, "Wrong status code for non empty chain in body")
+		// TODO(Martin2112): Remove not implemented from test when all the handlers have been written
+		assert.True(t, resp.StatusCode == http.StatusNotImplemented || resp.StatusCode == http.StatusBadRequest, "Wrong status code for GET to GET handler")
```

Comment:
It could be helpful to include the value of `resp.StatusCode` in the failure message. I expect once the "not implemented" part of that test is removed, this would just turn into `assert.Equal` though, which would achieve this too.